ivadomed/scripts/training_curve.py

```diff
@@ -43,10 +43,9 @@ def check_events_numbers(input_folder):
     Args:
         input_folder (str): Input folder path.
     """
-    for fold in os.listdir(input_folder):
-        fold_path = os.path.join(input_folder, fold)
-        if os.path.isdir(fold_path):
-            event_list = [f for f in os.listdir(fold_path) if f.startswith("events.out.tfevents.")]
+    for fold_path in Path(input_folder).iterdir():
+        if fold_path.is_dir():
+            event_list = [f.name for f in fold_path.iterdir() if f.name.startswith("events.out.tfevents.")]
```

Comment:
This is a good comfortable length to use list expansion. Anything more complicated, I woudl recommend breaking into smaller chunks IF loops.